src/platforms/android/migration.mdx

```diff
@@ -5,7 +5,7 @@ redirect_from:
   - /platforms/android/migrate/
 ---
 
-## Migrating from `sentry-android 2.x` to `sentry-android 3.x`
+## Migrating from `sentry-android 2.x` to `sentry-android 4.x`
```

Comment:
`m`: I would rather keep this and state that there is no migration needed between 3.x to 4.x.

Reply:
Agreed. Could even say that: The performance API was added but API-wise no breaking changes exist.

Also, I find it hard to believe there are 0 breaking changes :D

Reply:
Oh it's Android specific. I believe it now

Reply:
will just close it then